docs/design/autoware-architecture/planning/index.md

```diff
@@ -75,6 +79,22 @@ The following describes the input/output concept between Planning Component and
 - **To API Layer**
   - Planning factors: Provides information about the reasoning behind the current planning behavior. This may include the position of target objects to avoid, obstacles that led to the decision to stop, and other relevant information.
 
+### Internal interface in the planning component
+
+- **Mission Planning to Scenario Planning**
+  - Route: Offers guidance for the path that needs to be followed from the starting point to the destination. This path is determined based on information such as lane IDs defined on the map. At the route level, it doesn't explicitly indicate which specific lanes to take, and the route can contain multiple lanes.
+- **Behavior Planning to Motion Planning**
+  - Path: Provides a rough position and velocity to be followed by the vehicle. These path points are defined with an approximate interval of about 1 meter, although it's not strictly limited to that.
```

Comment:
The part after ", although ..." sounds a bit weird. How about adding a bit more details like the following:
```suggestion
  - Path: Provides a rough position and velocity to be followed by the vehicle. These path points are usually defined with an interval of about 1 meter. Although other interval distances are possible, it may impact the precision or performance of the planning component.
```